app/services/rapid_ready_for_decision/hypertension_special_issue_manager.rb

```diff
@@ -23,13 +23,16 @@ def add_special_issue
     RRD_CODE = 'RRD'
 
     def hypertension_increase?(disability)
-      RapidReadyForDecision::ProcessorSelector.disability_increase?(disability, HYPERTENSION_CODE)
+      # Does the same as RapidReadyForDecision::ProcessorSelector.disability_increase?(disability, HYPERTENSION_CODE)
+      # except using symbols as the keys instead of strings
+      disability[:diagnosticCode] == HYPERTENSION_CODE &&
+        disability[:disabilityActionType].downcase == 'increase'
     end
 
     # Must return an array containing special string codes for EVSS
     def add_rrd_code(disability)
-      disability['specialIssues'] ||= []
-      disability['specialIssues'].append(RRD_CODE) unless disability['specialIssues'].include?(RRD_CODE)
+      disability[:specialIssues] ||= []
+      disability[:specialIssues].append(RRD_CODE) unless disability[:specialIssues].include?(RRD_CODE)
```

Comment:
Reverts [this changes](https://github.com/department-of-veterans-affairs/vets-api/pull/9181/files#diff-a066d208bf973f8742bd9bf3d73ea2a300f06736bb42c948940c9cd8e2f5b5c8L36)